Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -167,12 +165,20 @@ public class NavigationCameraStateTransition: CameraStateTransition {
             animatorBearing.stopAnimation()
         }
         
-        let bearingTimingParameters = UICubicTimingParameters(controlPoint1: CGPoint(x: 0.0, y: 0.0), controlPoint2: CGPoint(x: 1.0, y: 1.0))
-        animatorBearing = mapView.camera.makeAnimator(duration: duration, timingParameters: bearingTimingParameters) { (transition) in
-            transition.bearing.toValue = bearing
+        // Check whether the bearing change is larger than a certain threshold when current camera state is following.
+        var updateCameraBearing: Bool = true
+        if state == .following {
+            updateCameraBearing = (abs(mapView.cameraState.bearing - bearing) >= 1.0)
         }
         
-        animatorBearing?.startAnimation()
+        if updateCameraBearing {
```

Comment:
`updateCameraBearing` only appears to be set once and then read in the following conditional statement. We can collapse this into one line.